packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -408,3 +421,48 @@ const generateMissingOrAliasedRequiredFields = ({
 
     return generateMissingOrAliasedFields({ fieldNames: requiredFields, selection });
 };
+
+function createFulltextProjection({
+    resolveTree,
+    node,
+    context,
+    chainStr,
+    varName,
+    literalElements,
+    resolveType,
+}: {
+    resolveTree: ResolveTree;
+    node: Node;
+    context: Context;
+    chainStr?: string;
+    varName: string;
+    literalElements?: boolean;
+    resolveType?: boolean;
+}): ProjectionResult {
+    if (!resolveTree.fieldsByTypeName[node.fulltextTypeNames.result][lowerFirst(node.name)]) {
```

Comment:
Question: Do we need to use `lowerFirst` here? Feels a bit flaky to depend on that logic to find the correct nodeResolveTree

Reply:
Yeah it's needed because the query field this is looking for has been created with `lowerFirst`

Reply:
Okay my bad, turns out I can just use `node.singular` instead